torch/_export/__init__.py

```diff
@@ -92,6 +92,8 @@ def export(
     f: Callable,
     args: Tuple[Any],
     constraints: Optional[List[Constraint]] = None,
+    *,
+    _turn_off_export_validity=False,
```

Comment:
Rename to `_add_runtime_assertions=True`? Or were you imagining other uses?

Also, in general, prefer "disable" over "turn_off"

Reply:
Sure sounds good